bindings/rust/wasmedge-sys/src/async_env.rs

```diff
@@ -0,0 +1,113 @@
+use std::{
+    cell::UnsafeCell,
+    future::Future,
+    pin::Pin,
+    ptr,
+    task::{Context, Poll},
+};
+use wasmtime_fiber::Fiber;
+pub(crate) struct Reset<T: Copy>(pub(crate) *mut T, pub(crate) T);
+
+impl<T: Copy> Drop for Reset<T> {
+    fn drop(&mut self) {
+        unsafe {
+            *self.0 = self.1;
+        }
+    }
+}
+
+pub(crate) struct FiberFuture<'a> {
+    fiber: Fiber<'a, Result<(), ()>, (), Result<(), ()>>,
+    current_poll_cx: *mut *mut Context<'static>,
+    // engine: Engine,
```

Comment:
Why does this comment out?

Reply:
Cause this was added in previous versions and not needed anymore, will delete it.